plugins/chat/assets/javascripts/discourse/components/chat-channel.js

```diff
@@ -1079,4 +1088,13 @@ export default class ChatLivePane extends Component {
     cancel(this._laterComputeHandler);
     cancel(this._debounceFetchMessagesHandler);
   }
+
+  async #tryToEnsureMentionsLoaded(message) {
+    try {
+      await message.ensureMentionsLoaded();
```

Comment:
How about we add a param `ensureMentionsLoaded`, something like `ignore_failure`, `false` by default, that way we don't have to repeat that code everywhere and we could also remove the comment.

```suggestion
      await message.ensureMentionsLoaded();
```

Reply:
A nice suggestion 👍, done.